ui/src/containers/Login.js

```diff
@@ -48,7 +49,24 @@ const LogoContainer = styled.div`
 `;
 
 const LoginForm = props => {
-  const { values, touched, errors, handleChange, isSubmitting, intl } = props;
+  const {
+    values,
+    touched,
+    errors,
+    dirty,
+    intl,
+    setFieldValue,
+    setFieldTouched,
+    asyncErrors
+  } = props;
+  //handleChange of the Formik props does not update 'values' when field value is empty
+  const handleChange = (e, field) => {
```

Comment:
You could make it nicer to use with currying:

```suggestion
  const handleChange = field => e => {
```

and below:

```javascript
<Input
  name="username"
  ...
  onChange={handleChange('username')}
/>
```